.github/workflows/CI.yml

```diff
@@ -116,17 +116,32 @@ jobs:
         with:
           command: fmt
           args: --all -- --check
-
   checker:
     name: gtk-rs checker
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: checker
+    strategy:
+      matrix:
+        crate:
+          - atk
+          - gdk
+          - gdk-pixbuf
+          - gdkx11
+          - gio
+          - glib
+          - graphene
+          - gtk
+          - pango
+          - pangocairo
     steps:
       - uses: actions/checkout@v2
         with:
           repository: gtk-rs/checker
           ref: master
           path: checker
-      - working-directory: checker
-        run: cargo build --release
-      - run: "./checker/target/release/checker ."
-      - run: "./checker/check_init_asserts"
+      - run: cargo build --release
```

Comment:
The `checker` is doing a lot of things, not just checking that it builds fine. So please don't remove it? :)

Reply:
No I didn't remove it ^^  the check_init_asserts is disabled for now until I fix the checker (it's still broken)